llvm/test/MC/Disassembler/X86/apx/rex2-bit.txt

```diff
@@ -0,0 +1,238 @@
+# RUN: llvm-mc -triple x86_64 -disassemble %s | FileCheck %s --check-prefix=ATT
+# RUN: llvm-mc -triple x86_64 -disassemble -output-asm-variant=1 %s | FileCheck %s --check-prefix=INTEL
+
+## R bit
+
+# ATT:   leal	(%rax), %r16d
+# INTEL: lea	r16d, [rax]
```

Comment:
Do we need a DWORD PTR for intel syntax?

Reply:
No, LEA is special b/c it does not read memory.